src/you_get/common.py

```diff
@@ -682,19 +690,21 @@ def __init__(self, total_size, total_pieces = 1):
     def update(self):
         self.displayed = True
         bar_size = self.bar_size
-        percent = round(self.received * 100 / self.total_size, 1)
-        if percent >= 100:
+        percent_str = round_str(self.received * 100 / self.total_size, 1)
+        if self.received == self.total_size:
             percent = 100
```

Comment:
Ah, I see.
The old one - 100.0% is too long for the console so it prints two lines
![2017-07-23 9 32 24](https://user-images.githubusercontent.com/1081266/28495689-e28e6c0e-6f89-11e7-91ba-4783213fa801.png)

The new one - make it 99.9 if ```percent_str``` is ```100.0``` but not completed
![2017-07-23 9 33 34](https://user-images.githubusercontent.com/1081266/28495699-25bad38c-6f8a-11e7-9975-5dc9e2c463d4.png)

We cannot use ```round``` here